### ISSUE TITLE
Cannot find module '../src/config'

### ISSUE THREAD

**Issue by iFlameing** [iFlameing/test-plone-addon#1]
@tiberiuichim I made a demo volto project and passing the addon as volto-slate but when I am building the demo it is giving me error can you please take a look.

/home/iflameing/test1/demo/build/server.js:4429
!(function webpackMissingModule() { var e = new Error("Cannot find module '../src/config'"); e.code = 'MODULE_NOT_FOUND'; throw e; }());
                                            ^

Cannot find module '../src/config'

**Comment by tiberiuichim** [comment]
@iFlameing For several reasons you should use the develop branch (we keep the master locked, to have another developer able to develop testing infrastructure around it) https://github.com/eea/volto-slate/tree/develop

Another thing, volto-slate is undergoing some refactoring right now. I'll cut a release on Monday. Until then, things may be unstable. You may have more luck with this release: https://github.com/eea/volto-slate/releases/tag/0.3.6

Also, it needs "special Volto", it requires at least 2 yet-unmerged pull requests in Volto. We use the "kitchen_sink_new" branch in this repo: https://github.com/eea/volto

**Comment by tiberiuichim** [comment]
Check this repo if you need to see how we launch the volto-slate. It's the repo I use to drive the volto-slate development. Note the use of yalc, but you don't have to use that, you can just use the branch of Volto that I've mentioned above.

**Comment by iFlameing** [comment]
@tiberiuichim Yeah!! I used the eea/volto and it worked. I found the problem with the current volto release is, it doesn't resolve the config
```js
"~/config"
```
 file properly if it is used by the add-on component.

for example this line
```js
import { settings } from '~/config';
```
https://github.com/eea/volto-slate/blob/master/src/TextBlock/TextBlockEdit.jsx#L12.
I don't know how Volto resolve this '~/config` file :(

**Comment by tiberiuichim** [comment]
@iFlameing In my last comment I meant this repo: https://github.com/eea/volto-slate-project/tree/develop, I forgot to paste, sorry

**Comment by tiberiuichim** [comment]
@iFlameing also, please use the "develop" branch from volto-slate.
